tests/candev/main.c

```diff
@@ -192,7 +192,7 @@ int main(void)
     puts("candev test application\n");
 
     isrpipe_init(&rxbuf, (uint8_t *)rx_ringbuf, sizeof(rx_ringbuf));
-#ifdef BOARD_NATIVE
+#ifdef CAN_DRIVER_CAN_NATIVE
```

Comment:
I think It would be better to use the automatic generated macros by the build system for the modules. Instead of creating your own CFLAG I would check:
```suggestion
#if IS_USED(MODULE_CAN_LINUX)
```

Then it can be extended for the other drivers when introduced.

Reply:
Not sure how it will work out in the future (I will reevaluate the change in the next PR. If it makes things a lot more complicated I might consider reverting the change).

But I applied your suggestion for now.